flexget/entry.py

```diff
@@ -290,3 +290,9 @@ def __hash__(self):
 
     def __repr__(self):
         return '<Entry(title=%s,state=%s)>' % (self['title'], self._state)
+
+    def to_dict(self):
```

Comment:
Do we really need this? Entry is already a subclass of dict, and if you want to put it in a normal dict you can do dict(entry)